### DISCUSSION TITLE
Lost my top languages stats after moving repositories to a organization

### DISCUSSION THREAD

**Discussion by VianaSamuel** [anuraghazra/github-readme-stats#2150] [Q&A] [answered]
Hey there.
Recently I've created a organization and moved some repositories of mine there.
I noticed that my top languages stats was changed by that, some languages were gone.
There's something I can do to undo this?
If I move the repositories from the organization to my profile again, will the stats come back again?

**Comment by rickstaa** [comment, marked as answer]
@VianaSamuel Currently, we do not include stats of organization repositories in the cards. However, this is one of the most requested features (see https://github.com/anuraghazra/github-readme-stats/issues/1). This feature has been implemented in https://github.com/anuraghazra/github-readme-stats/pull/1122, but some things must be fixed. However, you can already use this feature on your own fork by deploying your own Vercel instance (see https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance). You can follow the steps explained in https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 to do this. As a result, you don't need to move the repositories back to your own profile.

**Reply by VianaSamuel** [reply]
That's great! Thanks.

**Reply by rickstaa** [reply]
@VianaSamuel great, you can check it out [my profile](https://github.com/rickstaa/rickstaa) to see the behaviour in action. 👍🏻 Could you please accept my answer so that this question is marked as solved in the Discussion interface :nerd:?

**Reply by VianaSamuel** [reply]
Sorry, I had forgotten! Thanks again.
